apps/frontend/src/components/chat/ChatPane.tsx

```
@@ -1,88 +1,89 @@
-          <GitBranch size={16} />
-          <span className="text-sm">Fork this thread</span>
-        </button>
-      </div>
+	return (
+		<div className="flex-1 flex flex-col bg-gray-50 relative">
+			<div className="flex items-center justify-between px-4 py-2 bg-white border-b border-gray-200">
+				<h2 className="font-medium text-gray-800">{currentThread.name}</h2>
+				<button
+					className="flex items-center space-x-1 text-gray-700 hover:text-blue-700 transition-colors px-2 py-1 rounded hover:bg-gray-100"
+					onClick={() => setIsCreatingBranch(true)}
+				>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)